### PR TITLE
Contribute data: Upload files

### PR DESCRIPTION
**Pull request by mbarrenechea**
This PR allows users to upload files

### CODE REVIEW

**Comment by mbarrenechea** [comment]
image

File: app/assets/javascripts/static/ContributeDataView.js [hunk 1]
```diff
@@ -81,6 +86,63 @@ define([
...
+
+    setUploadField: function() {
+      var that = this;
+
+      this.$fieldFileUpload.fileupload({
+        url: '/data/upload',
+        dataType: 'json',
+        autoUpload: true,
+        // acceptFileTypes: /(\.|\/)(gif|jpe?g|png)$/i,
+        maxFileSize: 10000000, // 10 MB
```
Comment (geriux) [inline app/assets/javascripts/static/ContributeDataView.js]: We should start using constants for things like the max file size, so it's easier to see if its at the top of the view.

File: app/assets/javascripts/stories/views/StoriesNewView.js [hunk 2]
```diff
@@ -256,7 +256,7 @@ define([
             var confirmation = confirm('Are you sure?')
 
             if (confirmation == true) {
```
Comment (geriux) [inline app/assets/javascripts/stories/views/StoriesNewView.js]: Let's try to use === 😁

**Comment by geriux** [comment]
I think we should redirect the user to the home page or another section when the form is submitted, it gave me the impression that it didn't work, other than that it works great! 😁

**Pull request merged**